src/js/tech/html5.js

```diff
@@ -224,9 +224,32 @@ class Html5 extends Tech {
     // for these scenarios.
     const SAFE_TIME_DELTA = TIME_FUDGE_FACTOR * 3;
 
+    // try to determine whether a successful playback occurred
+    const SUCCESSFUL_PLAYBACK_TIME = 1;
+
+    let successfulPlayback = false;
+
+    const timeUpdateListener = function() {
+      if (this.currentTime() > SUCCESSFUL_PLAYBACK_TIME) {
+        this.off('timeupdate', timeUpdateListener);
+        successfulPlayback = true;
```

Comment:
Might be better to move the ios check into a helper method, with a way to disable it. Then enable it here on successful playback and disable it below on new video.

Reply:
I would agree with that.

Reply:
Thanks for the feedback. Will adjust this pull request.

Reply:
I've adjusted the pull request and tested it locally in combination with videojs-ima on iOS.